src/screens/SendToken/SendTokenPinConfirmScreen.js

```diff
@@ -98,11 +98,11 @@ class SendTokenPinConfirmScreen extends React.Component<Props, State> {
   };
 
   navigateToTransactionState = (params: ?Object) => {
-    const { navigation } = this.props;
+    const { navigation, accounts } = this.props;
     const { transactionPayload } = this.state;
     const transactionType = navigation.getParam('transactionType', '');
 
-    navigation.navigate(SEND_TOKEN_TRANSACTION, { ...params, transactionPayload, transactionType });
+    navigation.navigate(SEND_TOKEN_TRANSACTION, { ...params, transactionPayload, transactionType, accounts });
```

Comment:
Seems a bit odd that we're passing a general selector (i.e. not related to transaction to be displayed) result in navigation params. Pls query accounts in SendTokenTransaction screen instead.